packages/sections/src/common/Literature/LiteratureContext.tsx

```diff
@@ -0,0 +1,120 @@
+import isEmpty from "lodash/isEmpty";
+import { createContext, useContext, useReducer } from 'react';
+import type { LiteratureStateType, DetailsStateType } from './types';
+import { getPage } from "ui";
+
+export const defaultLiteratureState: LiteratureStateType = {
+  id: "",
+  cursor: "",
+  category: ["disease", "drug", "target"],
+  query: null,
+  globalEntity: null,
+  entities: [],
+  selectedEntities: [],
+  startYear: null,
+  startMonth: null,
+  endYear: null,
+  endMonth: null,
+  earliestPubYear: 0,
+  litsIds: [],
+  page: 0,
+  pageSize: 5,
+  litsCount: 0,
+  loadingEntities: false,
+};
+
+const LiteratureContext = createContext(null as any);
+const LiteratureDispatchContext = createContext(null as any);
```

Comment:
For typing the actions, you can use the same pattern of AotF FocusState https://github.com/opentargets/ot-ui-apps/blob/4e9fc2e5e25e2cccd32f338aad99d77be59b0f6a/apps/platform/src/components/AssociationsToolkit/context/AssociationsFocusContext.tsx#L18